### PR TITLE
Create press-kit.md

### PR DESCRIPTION
**Pull request by InessaPawson**
This is a very abbreviated version of the future NumPy press kit. Need to add a link to the hi-res NumPy logo.

### CODE REVIEW

File: content/en/press-kit.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Press kit
+sidebar: false
+---
+
+We would like to make it easy for you to include the NumPy brand identity in your next academic paper, course materials or presentation. 
+
+You will find a high resolution logo of NumPy [here](https://numpy.org).
```
Comment (rgommers) [inline content/en/press-kit.md]: For now let's link to https://github.com/numpy/numpy/tree/master/branding/icons, I don't think we have anything higher resolution. Once we have a new logo, we can update this.

File: content/en/press-kit.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Note that by using the numpy.org resources, you accept [NumPy Code of Conduct](https://numpy.org/code-of-conduct).
```
Comment (rgommers) [inline content/en/press-kit.md]: minor: suggest changing to "... accept the [NumPy ..."

Comment (InessaPawson) [reply content/en/press-kit.md]: Done. Thank you for catching it, @rgommers!

File: content/en/press-kit.md (hunk 1 above)
Comment (InessaPawson) [reply content/en/press-kit.md]: Works for me!

**Comment by rgommers** [comment]
Merged, thanks @InessaPawson

**Pull request merged**